charts/operator/templates/role.yaml

```diff
@@ -391,11 +381,6 @@ rules:
       - topics
     verbs:
       - create
-  - apiGroups:
-      - ""
-    resources:
-      - persistentvolumeclaims
-    verbs:
```

Comment:
Why this is removed? Decommission capabilities will always be available as a side car? I thought it will be an option THIS or THAT.

Reply:
i have no idea what you mean. ill revert this file since there is no harm and I cannot remember why.

Reply:
This was not supposed to be here, only RP changes are supposed to be here.